core/src/test/kotlin/io/sharing/server/core/product/domain/ProductTest.kt

```diff
@@ -50,6 +50,46 @@ class ProductTest {
             "image6", "image7", "image8", "image9", "image10"
         )
     }
+
+    @Test
+    fun `상품 상태 이용 불가능한 상품으로 변경`() {
+        val product = createProduct()
```

Comment:
UNAVAILABLE 상태가 아니었다는것을 명시해두는 세팅을 해주는편이 더 좋을거같습니다
```suggestion
        val product = createProduct().apply { status = AVAILABLE }
```